frontend/apps/filemanager/filemanagerfilesearcher.lua

```diff
@@ -206,9 +228,16 @@ function FileSearcher:isFileMatch(filename, fullpath, search_string, is_file)
 end
 
 function FileSearcher:showSearchResultsMessage(no_results)
-    local text = no_results and T(_("No results for '%1'."), self.search_string)
+    local text = no_results and T(_("No results for '%1'."), FileSearcher.search_string)
     if self.no_metadata_count == 0 then
-        UIManager:show(InfoMessage:new{ text = text })
+        UIManager:show(ConfirmBox:new{
+            text = text,
+            icon = "notice-info",
+            ok_text = _("New search"),
```

Comment:
I would suggest `Edit search` or `Modify search`. "New" implies a blank slate.

Reply:
Any other variants? I don't feel excited with the proposed ones.

Reply:
Continue search? Search again? Just a magnifying glass icon?